lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -301,6 +301,33 @@ function dosomething_helpers_format_data($data) {
 }
 
 /**
+ * Utility funciton to get base URL.
+ *
+ * @return string
+ */
+function dosomething_helpers_get_base_url(){
```

Comment:
I always love the idea of DRY'ing up code, but I sorta feel like these helpers are a little redundant. 

For example:

* `dosomething_helpers_get_base_url()` replaces the `url()` call which is already a helper that drupal provides. So in these cases we would really just be replacing one line of code with the other. It does make it a little bit more readible. One way of maybe making this more helpful is to take in the `$language_prefix` as a parameter so that it can be more dynamic? 

* A function like `dosomething_helpers_get_country_prefix()` already exists in `dosomething_global_get_current_country_code()` so I'm not sure we need this here as well. It might not have existed when we initially did share stuff so that it why it wasn't used. But totally understand the confusion there.

* `dosomething_helpers_get_current_url_path()` really just replaces one line of code with another. 

One thing we could think about is a helper that does all of [this] (https://github.com/DoSomething/phoenix/blob/bb4fa06179e98a3b76cc14dd5a4791edfc8cf3e1/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc#L129-L154) in one function since you have to sorta follow all these steps when making the share bar...but a lot of the "steps" you need are specific to each configuration of the bar so maybe not.